src/RegExpr.pas

```diff
@@ -1611,7 +1633,18 @@ procedure TRegExpr.SetModifier(AIndex: integer; ASet: boolean);
 
 function TRegExpr.IsWordChar(AChar: REChar): boolean;
 begin
+  {$IFDEF UseWordChars}
   Result := Pos(AChar, fWordChars) > 0;
```

Comment:
Here we check with fWordChars if UseWordChars.
But that never happen - because we do not use OP_ANYLETTER in this case - see below.
So if UseWordChars we emit to opcode full set fWordChars and won't use IsWordChar at all.

Reply:
"если юзается UseWordChars то  ANYLETTER не эмитится и IsWordChars не юзается". ну да, я про это спрашивал уже. вроде. или юзается все-таки? там два места в коде EmitNode(ANYLETTER)- одно все равно дергает IsWordChar

Reply:
В общем, пач ничего не портит

Reply:
да верно - эта странность в коде просто есть и патч ничего в этом не меняет